internal/docs/jpa/PersistenceSpecAssertions.html

```diff
@@ -3,7 +3,7 @@
 <META http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <!--
 
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This should be 2018, 2020. Please check for such cases.

Reply:
@alwin-joseph , thanks for pointing it out, I'll check for these cases thoroughly.